src/scripts.ts

```diff
@@ -2,18 +2,17 @@ import * as _ from "lodash";
 import * as fs from "fs";
 import {EventEmitter} from "events";
 import * as log4js from "log4js";
-import * as Linter from "tslint";
-import {LintResult} from "tslint/lib/lint";
+import {ILinterOptions, LintResult, Linter} from "tslint";
 import * as karma from "karma";
 
 import {ScriptsLintingConfig, ScriptsTestConfig} from "./config/config.interface";
 import * as utils from "./utils";
 
 export class ScriptLinter extends EventEmitter {
 
-  private configuration = JSON.parse(fs.readFileSync("tslint.json", "utf8"));
+  private configuration                               = JSON.parse(fs.readFileSync("tslint.json", "utf8"));
```

Comment:
Those spacings are inconsistent with the rest of this project. I would vote for not having them.